rootfs/builder/build.sh

```diff
@@ -189,6 +189,14 @@ else
     tar -z --exclude='.git' -C $build_root -cf $slug_file . | cat
 fi
 
+if [[ ! -f "$build_root/Procfile" ]]; then
+	if [[ -s "$build_root/.release" ]]; then
+		ruby -e "require 'yaml';procTypes = (YAML.load_file('$build_root/.release')['default_process_types']);open('$build_root/Procfile','w') {|f| YAML.dump(procTypes,f)}"
```

Comment:
@kmala good point, but the builder create it before it creates and uploads the tar?